impeller/entity/contents/clip_contents.cc

```diff
@@ -80,59 +80,59 @@ bool ClipContents::Render(const ContentContext& renderer,
 
   VS::FrameInfo info;
 
-  Command cmd;
-
   auto options = OptionsFromPass(pass);
   options.blend_mode = BlendMode::kDestination;
-  cmd.stencil_reference = entity.GetClipDepth();
+  pass.SetStencilReference(entity.GetClipDepth());
   options.stencil_compare = CompareFunction::kEqual;
   options.stencil_operation = StencilOperation::kIncrementClamp;
 
   if (clip_op_ == Entity::ClipOperation::kDifference) {
     {
-      DEBUG_COMMAND_INFO(cmd, "Difference Clip (Increment)");
+      pass.SetCommandLabel("Difference Clip (Increment)");
 
       auto points = Rect::MakeSize(pass.GetRenderTargetSize()).GetPoints();
       auto vertices =
           VertexBufferBuilder<VS::PerVertexData>{}
               .AddVertices({{points[0]}, {points[1]}, {points[2]}, {points[3]}})
               .CreateVertexBuffer(renderer.GetTransientsBuffer());
-      cmd.BindVertices(std::move(vertices));
+
+      pass.SetVertexBuffer(std::move(vertices));
 
       info.mvp = pass.GetOrthographicTransform();
-      VS::BindFrameInfo(cmd,
+      VS::BindFrameInfo(pass,
                         renderer.GetTransientsBuffer().EmplaceUniform(info));
 
       options.primitive_type = PrimitiveType::kTriangleStrip;
-      cmd.pipeline = renderer.GetClipPipeline(options);
-      pass.AddCommand(Command(cmd));
+      pass.SetPipeline(renderer.GetClipPipeline(options));
+      pass.SetStencilReference(entity.GetClipDepth());
```

Comment:
Is this difference intentional?  This used to be set outside of the conditional.

Reply:
It shouldn't actually matter, this one is redundant since I call Draw immediately below.